src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -79,6 +81,21 @@ private void handleCommandEntered() {
         }
     }
 
+    /**
+     * Handles the View Selection.
+     */
+    public void handleViewSelected(MultipleSelectionModel<Person> selectedPersonModel) {
+        int index = selectedPersonModel.getSelectedIndices().get(0);
```

Comment:
Will this violating the Law of Demeter?

Reply:
yes i think it might, but is unavoidable in this case.